src/Shell/Helper/TableHelper.php

```diff
@@ -46,6 +46,7 @@ protected function _calculateWidths(array $rows): array
         $widths = [];
         foreach ($rows as $line) {
             foreach (array_values($line) as $k => $v) {
+                $v = is_integer($v) ? (string)$v : $v;
```

Comment:
What about numeric values that would be castable?
I wonder if it should just always cast.
If it doesnt work it would thow an exception anyway.

Reply:
I was unsure whether that was the right thing, but I agree. Have committed a change.

Reply:
Always casting threw a new wrench in the works. Should we resolve the psalm issue by removing the nullable string parameter?

Reply:
> Always casting threw a new wrench in the works. Should we resolve the psalm issue by removing the nullable parameter text in the _cellWidth method? That also means replacing the null checks with strlen checks instead.

That sounds like a good idea to me.